src/controller/CHIPDevice.cpp

```diff
@@ -75,7 +114,8 @@ CHIP_ERROR Device::SendMessage(System::PacketBufferHandle buffer, PayloadHeader
         resend = buffer.CloneData();
     }
 
-    CHIP_ERROR err = mSessionManager->SendMessage(mSecureSession, payloadHeader, std::move(buffer));
+    CHIP_ERROR err =
+        exchange->SendMessage(protocolId, msgType, std::move(buffer), Messaging::SendFlags(Messaging::SendMessageFlags::kNone));
```

Comment:
That seems like it would be pretty common.  Can you please file a followup to have the flags arg be optional, with that value as default?

Reply:
https://github.com/project-chip/connectedhomeip/issues/5670